### PR TITLE
Adding a API for getting the curent offset to facilitate checkpointin…

### PR DESCRIPTION
**Pull request by raviorteja**
Adding a getOffset API to facilitate checkpointing systems to use the parso library.

### CODE REVIEW

File: src/main/java/com/epam/parso/SasFileReader.java [hunk 1]
```diff
@@ -54,4 +54,11 @@
      * @return the object of the {@link SasFileProperties} class that stores file metadata.
      */
     SasFileProperties getSasFileProperties();
+    
+    /**
+     * The function to return the index of the current row when reading the file sas7bdat file.
```
Comment (Yana-Guseva) [inline src/main/java/com/epam/parso/SasFileReader.java]: Please remove redundant 'file' word here and below.

Comment (raviorteja) [reply src/main/java/com/epam/parso/SasFileReader.java]: this is corrected, thank you.

**Comment by Yana-Guseva** [comment]
Could you please provide an example when this API will be helpful? Parso always requires to read the input file from the begining and it is not possible for it to read the source file from specified row. Just wondering how this can be used. Thank you.

**Comment by rifat-khanam** [comment]
> Could you please provide an example when this API will be helpful? Parso always requires to read the input file from the begining and it is not possible for it to read the source file from specified row. Just wondering how this can be used. Thank you.

We are using epam parso in Streamsets Data collector. in that records are processing in batches (1000 records/rows per batch) , to generate a new batch it requires file offset ( index points till where it reads lasts) so that in next batch it can start reading from 1001th row .

**Pull request merged**